internal/adding-a-new-field-readme.md

```diff
@@ -1,60 +1,73 @@
 
-* Add the new field to the storage protobuf 
-  * storage protobufs are under: `internal/proto/controller/storage`
-
-  * Define a gorm tag for the new field via `@gotags` (`@inject_tag` has been deprecated)
-
-  * Define a `custom_options.v1.mask_mapping` for the field which maps the storage `this` field to the API `that` field (yes, it's the opposite of how it's defined for the API protobuf)
+* Add the new field to the storage protobuf
 
+  * storage protobufs are under: `internal/proto/controller/storage`
+  * Define a gorm tag for the new field via `@gotags`
+    (`@inject_tag` has been deprecated)
+  * Define a `custom_options.v1.mask_mapping` for the field
+    which maps the storage `this` field to the API `that` field
+    (yes, it's the opposite of how it's defined for the API protobuf)
   * Run `make proto` after modifying storage protobuf
 
-* Extend the existing repository function for Updating the resource to incorporate the new field. This could/may entail defining new options for the Update function.
-  
-* Check the repository's Create function for the resource, since you may need to incorporate the new field here as well.
+* Extend the existing repository function for Updating the resource to incorporate the new field.
+  This could/may entail defining new options for the Update function.
+* Check the repository's Create function for the resource,
+  since you may need to incorporate the new field here as well.
 
 ## Add new fields to the API/SDK resource protobufs
-Now that the repository supports the new field, you can move on to adding this new field to Boundary's API layer and the generated SDK.  The first step in the process is to add it to the API/SDK protobufs.
 
-* API protobufs are under: `internal/proto/controller/api/resources`
-  
-* Define a `custom_options.v1.mask_mapping` tag for the field which maps the API `this` field to the storage `that` field (yes, it's the opposite of how it's defined for a storage protobuf)
+Now that the repository supports the new field,
+you can move on to adding this new field to Boundary's API layer and the generated SDK.
+The first step in the process is to add it to the API/SDK protobufs.
 
+* API protobufs are under: `internal/proto/controller/api/resources`
+* Define a `custom_options.v1.mask_mapping` tag for the field
+  which maps the API `this` field to the storage `that` field
+  (yes, it's the opposite of how it's defined for a storage protobuf)
 * Define a `custom_options.v1.generate_sdk_option` tag for the SDK
+* Define a data classification/filter tag for the field via `@gotags` 
+  which specifies how sensitive/secret/public data
+  will be handled for the API's audit events.
+  See the [classification rubric](./classification-rubric.md)
+  for guidelines on classification.
+  Please write unit tests to verify the audit event is properly "redacted"
+  (see the unit tests of `TestAuthenticate_Tags` and `TestAuthenticate` for examples).
```

Comment:
Can we make this link to the tests?

Reply:
Yea that is a good idea. I might need to wait until your PR is done though, since I think you will be moving the location of these tests.